src/pages/wallet-overview/wallet-overview.spec.ts

```diff
@@ -130,10 +145,25 @@ describe("WalletOverviewPage", () => {
     });
   })
 
-  it("should return false after deleting a wallet that doesn't exist", () => {
-    let result = comp.deleteSelectedWallet(null);
+  it("should open landing page and show a toast after deleting the only wallet", (done) => {
+    // Get all 3 wallets
+    comp.getAllWallets().then(data => {
```

Comment:
This test is dependent on `getAllWallets` working properly. I think it would be better to mock this. Best would probably be to set the `wallets` property manually from within this test. E.g.:

```
comp.wallets = [{...}, {...}, {...}]
```